### ISSUE TITLE
Marionette & mixen don't play nicely: this._configure is not a function

### ISSUE THREAD

**Issue by ericanderson** [HubSpot/mixen#2]
```
class Foo extends Mixen(Backbone.Marionette.ItemView)

new Foo()
```

**Comment by zackbloom** [comment]
I've replicated this, looking into it.

**Comment by ericanderson** [comment]
Marionette isn't coffeescript, which is likely the problem.

I've been thinking about ways to make Mixen work regardless of source language, not sure if that is good for Mixen (will likely add some bloat) or better for a different project.

**Comment by zackbloom** [comment]
That shouldn't be a problem.  I actually have a test case which reproduces this just using fake classes.  It's some interaction with how Backbones extend works.  I'll get the test case passing and the issue should be fixed.

I'm very interested to hear what issues you run into using it with js.

**Comment by tuxracer** [comment]
@zackbloom I've run into a similar problem with Chaplinjs. What seems to solve the problem is always adding backbone at the end of the mixen statement. Even though Chaplin.Collection is already extending Backbone.Collection

``` coffeescript
module.exports = class Collection extends Mixen Cache, Chaplin.Collection, Backbone.Collection
```

@ericanderson With marionette try:

``` coffeescript
class Foo extends Mixen Backbone.Marionette.ItemView, Backbone.View

new Foo
```

**Comment by zackbloom** [comment]
My test case is fixed in [v0.5.1](https://github.com/HubSpot/mixen/releases/tag/v0.5.1), please let me know if it still appears.
